cron.go

```diff
@@ -43,31 +54,12 @@ type Job struct {
 	Name string
 	// Cron-formatted rhythm (ie. 0,10,30 1-5 0 * * *)
 	Rhythm string
-	// Routine method
-	Func func(context.Context) error
-
-	Repeats  int32
-	DueTime  string
-	TTL      string
-	Data     *anypb.Any
-	Metadata map[string]string
-}
-
-func (j Job) Run(ctx context.Context) error {
-	return j.Func(ctx)
-}
-
-var (
-	nonAlphaNumerical = regexp.MustCompile("[^a-z0-9_]")
-)
-
-func (j Job) canonicalName() string {
-	return strcase.ToSnake(
-		nonAlphaNumerical.ReplaceAllString(
-			strings.ToLower(j.Name),
-			"_",
-		),
-	)
+	// The type of trigger
+	Type string
```

Comment:
What is type?

Reply:
Anything that the client understands BEFORE it tries to parse the payload.

Reply:
Anything that the client app understands and wants to read BEFORE parsing the payload object. It can be helpful to differentiate between service invocation, reminder or any different type of trigger on the app side.

Reply:
Please add a comment describing this.

Reply:
It was described already. Added some more wording to it.